apps/reward-manager/src/util.ts

```diff
@@ -0,0 +1,66 @@
+import { BigNumber } from "ethers";
```

Comment:
If these are shared methods between the reward bot and this sNOTE reinvestment bot then maybe we can refactor them into the `@notional-finance/utils` package or something.

Reply:
Do I also need to increase semver of util package and publish it to npm now that I added two new functions?